sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -860,9 +872,10 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
       response.parsedBody == undefined ||
       (Array.isArray(response.parsedBody) && response.parsedBody.length == 0)
```

Comment:
In the corresponding .Net code, the check is made for the status code 204 for `NoContent` to determine the case of message entity not found. Any reason we can't do the same instead of checking whether `response.parsedBody` is undefined or empty array?